### ISSUE TITLE
Adjust Report Data Error form for accessibility

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/MyopicVicar#1042]
Not mentioned specifically in Shaw Report, but form shares similar issues with other public forms.

**Comment by AlOneill** [comment]
Have a question about the preamble, which currently is:

"We are sorry that you feel the transcription may be in error. Please tell us why you believe we have misinterpreted the original record. We will adjust the record if, upon review of the original, we agree with your interpretation.
**Please be aware we do not adjust transcriptions based upon sources of information other than the original register.**"

@edickens @Captainkirkdawson @Sherlock21  :
This *seems* to imply that we are not interested in correcting **our** typos in transcribing a transcript source. Is that the intention? (Perhaps "original register" does not mean what it seems to mean ...)

**Comment by edickens** [comment]
We get emails saying that something is wrong but not providing any
    evidence other than they have seen it on Ancestry or FamilySearch. 
    So we do not accept any corrections without sight of an image of the
    original register.  Even Bishops transcripts can be wrong.  E
    
      On 17/12/2016 16:48, AlOneill wrote:
    
    
      Have a question about the preamble, which currently is:
      "We are sorry that you feel the transcription may be in error.
        Please tell us why you believe we have misinterpreted the
        original record. We will adjust the record if, upon review of
        the original, we agree with your interpretation.
        Please be aware we do not adjust transcriptions based
          upon sources of information other than the original register."
      @edickens @Captainkirkdawson @Sherlock21 :
        This seems to imply that we are not interested in
        correcting our typos in transcribing a
        transcript source. Is that the intention? (Perhaps "original
        register" does not mean what it seems to mean ...)

**Comment by Captainkirkdawson** [comment]
As usual you are correct. it likely should read, ..... original document we
transcribed.

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by AlOneill** [comment]
Thanks, Kirk.
Here is my suggestion:

"Thank you for reporting a possible error in our transcription. If we agree with your interpretation of the **original document we transcribed**, we will change our record."

**Comment by Captainkirkdawson** [comment]
How about

Thank you for reporting a possible error in our transcription. We will
change our record If we agree with your interpretation of the *original
document.*

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by AlOneill** [comment]
Fair enough to put the if-clause second, but think we need to retain the notion of the source of *that* particular transcription. "Original" turns out to be ambiguous, again! So:

"Thank you for reporting a possible error in our transcription. We will change our record if we agree with your interpretation of the **source document we transcribed**.

**Comment by Captainkirkdawson** [comment]
Good compromise

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by edickens** [comment]
The only thing about this now is that it does not allow for the 
researcher to send an images of the original registration. Sometimes 
(quite often) we no longer have access to the register image, but it the 
researcher has seen it (they may have visited the RO or Church) and sent 
us a copy, we will accept that.  So that is not "the source document we 
transcribed".  Also, if we have uploaded someone else's transcription, 
or a BT, then the researcher may be sending us more accurate 
information.  We accept that.  E

**Comment by edickens** [comment]
PS.  The main point is that they need to provide evidence, and that 
evidence must be an original entry, not a transcription.  E

**Comment by Sherlock21** [comment]
I think any change in our Transcription must not be based on a researcher's family knowledge or on a different document to that which we used as our source.

So I agree with Kirk's suggested text 3 items above here.

**Comment by PatReynolds** [comment]
If we transcribe (correctly) the Bishop's Transcript (for example), but a researcher sends us an image of the PR, showing that an error crept into the BT, I think we should NOT ammend the BT transcription, but we should transcribe the PR, so that future researchers will be able to understand why others may include the 'wrong' data on their family tree.  I think we have already decided that we will add notes if we don't have the image.

So the contact page should say something along the lines of: "Thank you for reporting a possible error in our transcription. We will change our record if we agree with your interpretation of **the source document we transcribed**.  If you have access to **a different document** which has different evidence, please provide details so that we can attach a note to our transcription.  We would be grateful if you could send us an image of the different document so that we may transcribe it.

**Comment by AlOneill** [comment]
@PatReynolds 
That seems eminently logical. 
How would it sit with the conditions described in
http://www.freereg.org.uk/cms/information-for-transcribers/permission-to-transcribe

Those details are probably inappropriate for presenting to a Researcher  -- but I would like to also update the Coord instructions for dealing with possible data errors so that everyone is clear what to do and how to respond in just such a situation.

**Comment by PatReynolds** [comment]
Hmm.. add for Researchers: " We would be grateful if you could send us an image of the different document so that we may transcribe it. Please note, some Record Offices and some websites allow you to make copies only for personal use - if you made a copy of a document under such restriction, we cannot use it."

**Comment by Captainkirkdawson** [comment]
Alison
Will there be any css changes or other changes for this page. All I seem to have at this time is the test change

**Comment by AlOneill** [comment]
Yes, there will be other changes: I have sorted out in html. Had a brief look at the erb file, but have not had a chance to do the work on it yet.

(Did not get notified of your prev comment. Have now restarted browser -- we shall see.)

**Comment by AlOneill** [comment]
When I submitted a **Data Error Report** with required fields not completed, I was surprised to get a **Contact Us** form returned.

Is this the expected behaviour??

**Comment by Captainkirkdawson** [comment]
Not sure I understand question.

The Data Error Report uses the Contact system in a specific manner with
certain values preset

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by AlOneill** [comment]
Cannot repeat to double-check -- getting a 503 message when I click on link from Record Detail page. The error messages for the form were returned on a "Contact Us" page, rather than a "Data Problem Report" page, and a quick look at the hidden fields showed little evidence that I was attempting to report a possible transcription error.

Could this explain why some users appear to use the wrong form when reporting a possible transcription error?

**Comment by Captainkirkdawson** [comment]
I have colobus out of service at this point working on trying to resolve
1049.

Kirk Dawson
5220 Riverside Drive
Fairmont Hot Springs, B.C.
V0B 1L1

**Comment by Captainkirkdawson** [comment]
Deployed code changes from @AlOneill onto test3
